packages/core/src/actions/invoke.ts

```diff
@@ -62,10 +62,7 @@ export function invoke<
         type,
         params: {
           ...params,
-          id: params.id,
-          src: params.src,
-          ref: new ObservableActorRef(behavior, id),
-          meta
```

Comment:
all removed properties are already part of the `params` so they were "set" here twice